app/views/exhibits/tech-nostalgia.md

```diff
@@ -44,7 +44,129 @@ John Campopiano
+
+### Conclusion
+
+In this paper my goal was to introduce the topic of tech-nostalgia, provide examples of how it has manifested itself in contemporary popular culture and the audiovisual arts, and highlight certain issues to examine how these ideas might erode our once steadfast adherence to certain preservation and archival norms. Just as the forefathers of the computer could never have envisioned how far the technology could take us, so too could they not have predicted that certain groups of computer users would utilize this same technology as a way of looking to and, in some instances, experiencing elements of the past. The interest in recreating aesthetics of our past with technologies of our modern day is not only intriguing because of its socio-cultural implications, but also because it presents unique challenges to those involved in collecting, describing, preserving, and making available digital information.
+
+### Endnotes
+
+<a name="footnote-1"></a><sup>1</sup> “Technostalgia.” 8 Mar 2012. Web. 1 Aug 2013. [http://en.wiktionary.org/wiki/technostalgia](http://en.wiktionary.org/wiki/technostalgia)
+
+<a name="footnote-2"></a><sup>2</sup>Fisher, Mark. “What is Hauntology?” *Film Quarterly*. Vol. 66, No. 1, (Fall 2012): 16-24. Print.
+
+<a name="footnote-3"></a><sup>3</sup>Jurgenson, Nathan. “The Faux-Vintage Photo: Full Essay Parts I, II, and III.” *Cyborgology*. 14 May 2011. Web. 28 May 2013. [Thesocietypages.org/cyborgology/2011/05/12/the-faux-vintage-photo-full-essay-parts-i-ii-and-iii/](http://www.thesocietypages.org/cyborgology/2011/05/12/the-faux-vintage-photo-full-essay-parts-i-ii-and-iii/)
+
+<a name="footnote-4"></a><sup>4</sup>Tierney, John. “What is Nostalgia Good For? Quite a Bit, Research Shows.” New York Times, 8 July 2013. Web. 9 July 2013. [http://www.nytimes.com/2013/07/09/science/what-is-nostalgia-good-for-quite-a-bit-research- shows.html?pagewanted=1&adxnnl=1&adxnnlx=1382360092-IzpG0hRGvR8AZhDBNk8xdA](http://www.nytimes.com/2013/07/09/science/what-is-nostalgia-good-for-quite-a-bit-research- shows.html?pagewanted=1&adxnnl=1&adxnnlx=1382360092-IzpG0hRGvR8AZhDBNk8xdA)
```

Comment:
@caseyedavis12 : this line and the next there's a space character in the URL. We should have tests to catch this, but something to keep an eye out for.